modules/ROOT/pages/cloudhub-faq.adoc

```diff
@@ -32,54 +31,54 @@ An https://anypoint.mulesoft.com/#/signup[Anypoint Platform account] gives you a
 * xref:access-management::managing-your-account.adoc[Creating an Account]
 * xref:access-management::managing-permissions.adoc[Managing Accounts Roles and Permissions]
 
-=== Where can I get Anypoint Studio?
+== Where can I get Anypoint Studio?
 
 Anypoint Studio can be https://www.mulesoft.com/lp/dl/studio[downloaded here]. From Studio, you have ability to deploy directly to CloudHub or run your applications locally on an embedded Mule runtime engine (Mule).
 
-=== How much does CloudHub cost?
+== How much does CloudHub cost?
 
 For specific pricing, please reach out to your account executive, your account development representative or contact us in the https://support.mulesoft.com[Support Portal].
 
-=== What is a Worker?
+== What is a Worker?
 
 Applications that expose real time services or APIs run on one or more "workers". Each worker is an instance of Mule that runs your integration applications. See xref:cloudhub-fabric.adoc[CloudHub Fabric] for details of how workers can be shared or doubled to scale your application and provide high availability.
 
-=== What is an Integration Application?
+== What is an Integration Application?
 
 An integration application is one that integrates two or more applications or services. This may be as simple as synchronizing Salesforce data to a local database to creating a mashup using Google Apps, Facebook, and Salesforce. These applications perform functions by grabbing data from different sources and publishing it to a new location or as a REST or Web service. In the traditional 3 tier application model an integration app represents the data tier through connectivity to application data and the service tier through logic used to combine, transform and route this data to consumers.
 
-=== What is an Anypoint Connector?
+== What is an Anypoint Connector?
 
 Anypoint Connectors provide a simple and easy way to integrate with SaaS and Cloud APIs such as Salesforce, Facebook, AWS, PayPal, and then to use them as services within Mule. With these connectors you are then able to create orchestration that spans both SaaS and on-premises applications, using Mule.
 
-=== What is a Mule Flow?
+== What is a Mule Flow?
 
 A Mule flow defines how Mule  processes messages between different applications. A flow defines a series of processors that  act on a message triggered by an event in an application, a request to a Web or REST service or a scheduled trigger.
 
 
-=== What Java version are you using?
+== What Java version are you using?
 
 * Workers that use Mule 4.0.0 and newer run on JDK 8.
```

Comment:
Per MuleSoft Style Guide, versions need to be "earlier" and "later" (not newer and older).

Reply:
Thanks. I did not edit anything else in this topic - just added the new one (and demoted all the headings so they show up in the right nav bar). Sigh. I guess I can fix that :)